src/user-guide/running-workflows/reflow.rst

```diff
@@ -128,38 +128,26 @@ Special Case: Triggering ``n=0`` Tasks
 
    - Triggering a task with a submitted or running job has no effect
      (it is already triggered).
-   - Triggering other :term:`active tasks <active task>` e.g. (a waiting
-     task which is held) queues it to run in the same flow.
-   - Triggering an :term:`incomplete task` queues it to re-run in the same flow.
+   - Triggering other :term:`active tasks <active task>`, including
+     :term:`incomplete <output completion>` tasks with a :term:`final status`,
+     queues them to run in the same flow.
```

Comment:
I don't think "final status" actually comes into this one:

```suggestion
   - Triggering other :term:`active tasks <active task>`, including
     tasks with :term:`incomplete outputs <output completion>`,
     queues them to run in the same flow.
```

Reply:
I only said "including" - because incomplete final-status tasks are the main target for retriggering by far. But I'll accept your change...